src/domain/data-decoder/entities/schemas/data-decoded.schema.ts

```diff
@@ -1,5 +1,22 @@
 import { Schema } from 'ajv';
+import { z } from 'zod';
 
+export const DataDecodedParameterSchema = z.object({
+  name: z.string(),
+  type: z.string(),
+  // z.unknown() makes the property optional but it should be defined
+  value: z.custom<Required<unknown>>(),
+  valueDecoded: z
+    .union([z.record(z.unknown()), z.array(z.record(z.unknown()))])
+    .optional(),
```

Comment:
Is there a reason for just keeping this as `optional()` instead of defining it as `nullish().default(null)` as we do in other places?

Also, the second member of the union (`z.array(z.record(z.unknown()))`) implies the array should contain objects, is this correct?